samples/OpenIdConnectSample/Startup.cs

```diff
@@ -39,10 +37,8 @@ public void ConfigureServices(IServiceCollection services)
                 sharedOptions.SignInScheme = CookieAuthenticationDefaults.AuthenticationScheme);
         }
 
-        public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
```

Comment:
The templates suck, configuring logging from `Main()` feels much cleaner.
But okay, I'll revert this change :smile: